src/dmd/dmodule.d

```diff
@@ -429,6 +429,7 @@ extern (C++) final class Module : Package
         if (doHdrGen)
             hdrfile = setOutfile(global.params.hdrname, global.params.hdrdir, arg, global.hdr_ext);
         //objfile = new File(objfilename);
+        escapetable = new Escape();
```

Comment:
What's the advantage of using a pointer which requires dynamically allocate an object?

Reply:
It's not about pro and cons here, it's about the cpp version that does not allow a module to store a Escape as value so it's made like that.